public/locations-list-map.js

```diff
@@ -285,7 +285,26 @@ $(function () {
     });
 
     if (showMap) {
-      initMap(stateFilter);
+      // set up a callback function to complete map initialization
+      window.initMap = () => initMap(stateFilter);
+
+      // load map based on current lang
```

Comment:
Can we pull this out to a function for clarity with a comment explaining that it depends on window.initMap() as the callback and why we need it?